tests/e2e/env/docker-compose.yml

```diff
@@ -2,8 +2,7 @@ version: '3'
 
 services:
   wordpress:
-    build: ./wordpress-xdebug
-    image: wcp_e2e_wordpress
+    image: wordpress:5.8.3-php7.4
```

Comment:
I am ok with the change as well, only concern is whether updating this image as we bump versions would become an additional task for us. Any way to get the `latest` we support here?

Reply:
I don't think it's an issue @deepakpathania. We update the WP & WC versions before running tests.

Reply:
> We update the WP & WC versions before running tests.

Yes, that's right. Also, if using `latest`, I am not sure if we can run the downgrade with this command: 
https://github.com/Automattic/woocommerce-payments/blob/5d19129845e132624bb08cf8f7bb325803f04516/tests/e2e/env/setup.sh#L152-L155

Based on the description here https://developer.wordpress.org/cli/commands/core/update/, it's for `new versions`.